openedx_events/enterprise/data.py

```diff
@@ -0,0 +1,44 @@
+"""
+Data attributes for events within the architecture subdomain `enterprise`.
+
+These attributes follow the form of attr objects specified in OEP-49 data
+pattern.
+"""
+from datetime import datetime
+
+import attr
+
+@attr.s(frozen=True)
+class TrackingEvent:
+    """
+    License events to be put on event bus
+    TODO: figure out proper description
+
+    Arguments:
+        license_uuid (str):
+        license_activation_key (str):
+        previous_license_uuid (str):
+        assigned_date (str):
+        activation_date (str):
+        assigned_lms_user_id (str):
+        assigned_email (str):
+        expiration_processed (bool):
+        auto_applied (bool):
+        enterprise_customer_uuid (str):
+        enterprise_customer_slug (str):
+        enterprise_customer_name (str):
+        customer_agreement_uuid (str):
```

Comment:
Could you help me document this or point me to a place where I could get this info? @rgarber